prime-router/build.gradle.kts

```diff
@@ -109,6 +109,7 @@ tasks.test {
 
     // Set max parellel forks as recommended in https://docs.gradle.org/current/userguide/performance.html
     maxParallelForks = (Runtime.getRuntime().availableProcessors() / 2).takeIf { it > 0 } ?: 1
+    setForkEvery(1)
```

Comment:
What does this do?  Per documentation:
By default, Gradle automatically uses a separate JVM when executing tests, so changing this property is usually not necessary.